src/DDTrace/Integrations/Laravel/V4/LaravelProvider.php

```diff
@@ -99,6 +100,7 @@ public function boot()
 
         PDOIntegration::load();
         ElasticSearchIntegration::load();
+        MongoClientIntegration::load();
```

Comment:
Please rebase on master and you will see you have to add this to the list of autoloaded integrations